docs/developer-guide/syntax-reference.rst

```diff
@@ -1242,6 +1242,18 @@ Scalar functions
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | ROUND                  |  ``ROUND(col1)``                                                          | Round a value to the nearest BIGINT value.        |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| SPLIT                  |  ``SPLIT(col1, delimiter)``                                               | Splits a string into an array of substrings based |
```

Comment:
@JimGalasyn : Moving forward (you should be aware), we should always include 2-3 examples per function in our docs.  @spena's "docs" in his PR description is better than these changes to the Syntax Guide. ;-)

Reply:
For sure, I have an internal ticket tracking this: DOCS-1122.

Reply:
@miguno Should I add the examples for the split() in the syntax guide? I didn't add them because I wasn't sure how large the syntax guide could be.